hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleanPlanActionExecutor.java

```diff
@@ -128,6 +155,9 @@ HoodieCleanerPlan requestClean(HoodieEngineContext context) {
 
   @Override
   public Option<HoodieCleanerPlan> execute() {
+    if (!needCleaning(config.getInlineCleaningTriggerStrategy())) {
```

Comment:
does it matter really if this is inline or async cleaning? since it affects only the planning

Reply:
It does not affect, let me change the name.